### PR TITLE
2361 kafka microservice

### PR DESCRIPTION
**Pull request by Hexenon**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: integration/microservices/e2e/sum-kafka.spec.ts [hunk 1]
```diff
@@ -11,27 +11,11 @@ import { Test } from '@nestjs/testing';
...
-  }
-}
-@Catch()
-class RpcErrorFilter implements RpcExceptionFilter {
-  catch(exception: RpcException): Observable<any> {
-    console.log(exception);
-    return throwError(exception);
-  }
-}
+import * as async from 'async';
```
Comment (mkaufmaner) [inline integration/microservices/e2e/sum-kafka.spec.ts]: How about using Promise.all for concurrency instead of async?

Comment (Hexenon) [reply integration/microservices/e2e/sum-kafka.spec.ts]: Yeah, we can refactor that, i used this async parallel before, and promise.all before, they both do the same i think, but its more prettier use the Promise.all :D

Comment (mkaufmaner) [reply integration/microservices/e2e/sum-kafka.spec.ts]: Would much rather use Promises or Observables.  Also I don't think async is a nestjs npm dependency. If you could refactor to use promises, that would be great.

Comment (Hexenon) [reply integration/microservices/e2e/sum-kafka.spec.ts]: @mkaufmaner its done

**Comment by Hexenon** [comment]
@mkaufmaner this PR is ready to be merged, please do ! to check coveralls, unless you have more comments

**Pull request merged**